lib/CPU-performance-lib.robot

```diff
@@ -23,7 +23,7 @@ Get CPU Frequency MIN
 
 Get CPU Temperature CURRENT
     [Documentation]    Get current CPU temperature.
-    ${temperature}=    Execute Command In Terminal    sensors 2>/dev/null | awk -F '[+°]' '/Package id 0:/ {printf $2}'
+    ${temperature}=    Execute Command In Terminal    sensors 2>/dev/null | awk -F '[+°]' '/Core 0:/ {printf $2}'
```

Comment:
So in order to fix the test for Minnow you will break all other platforms?

Why not just simply read the output for sensors and grep by `Package id 0:`, if not found use `Core 0:` else use `Package id 0:`? 

@macpijan do we have any control over such methodology of work causing endless loops of broken tests?

Reply:
problem addressed: https://github.com/Dasharo/open-source-firmware-validation/pull/441/commits/917f329e834668466070c91cf7d9e008982089d4